api/src/pcapi/routes/backoffice/filters.py

```diff
@@ -79,6 +79,13 @@ def format_deposit_type(deposit_type: finance_models.DepositType) -> str:
             return "Aucune information"
 
 
+def format_active_deposit(booking: bookings_models.Booking) -> str:
+    if booking.deposit:
+        if not booking.deposit.expirationDate or booking.deposit.expirationDate > datetime.datetime.utcnow():
```

Comment:
Je n'avais pas prêté attention, mais quand tu formates un `Deposit`, alors tu passes un `Deposit`, il n'y a pas de raison de passer un objet `Booking` : 
```suggestion
def format_active_deposit(deposit: dinance_models.Deposit | None) -> str:
    if deposit:
        if not deposit.expirationDate or deposit.expirationDate > datetime.datetime.utcnow():
```